qa/suites/rbd/iscsi/cluster/fixed-3.yaml

```diff
@@ -1,4 +1,19 @@
 roles:
-- [mon.a, osd.0, osd.1, osd.2, a_gateway.0]
-- [mon.b, osd.3, osd.4, osd.5, b_client.0]
-- [mon.c, mgr.x, osd.6, osd.7, c_gateway.1]
+- - host.a
+  - mon.a
+  - mgr.y
+  - osd.0
+  - osd.1
+  - client.0
+  - ceph.iscsi.iscsi.a
+- - host.b
+  - mon.b
+  - mgr.x
+  - osd.2
+  - client.1
+- - mon.c
+  - mgr.z
+  - osd.3
+  - osd.4
```

Comment:
Why did the number of daemons change (5 OSDs instead of 8, 3 MGRs instead of 1)?

Reply:
@idryomov with cephadm we need a mgr on the same node as the first mon.
```
Traceback (most recent call last):
  File "/home/ideepika/teuthology/teuthology/contextutil.py", line 31, in nested
    vars.append(enter())
  File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/ideepika/src/github.com_ideepika_ceph_1805f4cf34daf77600c3b39bbbec56eff4802fe2/qa/tasks/cephadm.py", line 1384, in initialize_config
    raise RuntimeError('no mgrs on the same host as first mon %s' % first_mon)
RuntimeError: no mgrs on the same host as first mon a 
```